### PR TITLE
Change numpy import commands

### PR DESCRIPTION
**Pull request by Anthchirp**
Follow the convention of importing `numpy as np`

### CODE REVIEW

**Comment by graeme-winter** [comment]
Thank you for this, I think it’ll help new people and I agree it’s good to follow conventions

File: test/algorithms/shoebox/test_mask_overlapping.py [hunk 1]
```diff
@@ -105,7 +107,6 @@ def tst_non_overlapping(reflections, non_overlapping, image_size):
 
 def tst_overlapping(reflections, overlapping, adjacency_list, image_size):
```
Comment (graeme-winter) [inline test/algorithms/shoebox/test_mask_overlapping.py:108]: ISTR convention is we now call test functions `test_` not `tst_` - does this mean this test is missed by `pytest`?

Comment (Anthchirp) [reply test/algorithms/shoebox/test_mask_overlapping.py:108]: No, they are helper functions for one main `test()` function. This is pretty old code from pre-pytest days that was just directly converted. Nowadays we would lay it out differently.

Comment (graeme-winter) [reply test/algorithms/shoebox/test_mask_overlapping.py:108]: 👍

**Pull request merged**